airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -379,6 +379,8 @@ definitions:
       datetime_format:
         description: The format of the datetime
         type: string
+      cursor_datetime_format:
+        type: string
```

Comment:
can you add a description here. I think we should emphasize that this is related to how we format the date cursor from the response. When discussing how to best approach this, the concept of request/response format was mentioned so that could be an indicator on how we present the documentation.

`description: The format string used to transform the datetime value in the API response into the format used by the cursor`